### PR TITLE
zh-translation: docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst

### PR DESCRIPTION
**Pull request by vgbhfive**
…keeper_proxy_filter.rst

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #362 
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
-ZooKeeper proxy
+ZooKeeper 代理
 ===============
 
-The ZooKeeper proxy filter decodes the client protocol for
-`Apache ZooKeeper <https://zookeeper.apache.org/>`_. It decodes the requests,
-responses and events in the payload. Most opcodes known in
-`ZooKeeper 3.5 <https://github.com/apache/zookeeper/blob/master/zookeeper-server/src/main/java/org/apache/zookeeper/ZooDefs.java>`_
-are supported. The unsupported ones are related to SASL authentication.
+ZooKeeper 代理过滤器解码 `Apache ZooKeeper <https://zookeeper.apache.org/>`_ 的客户端协议。解码在负载中的请求、响应、事件。其中支持 `ZooKeeper 3.5 <https://github.com/apache/zookeeper/blob/master/zookeeper-server/src/main/java/org/apache/zookeeper/ZooDefs.java>`_  中已知的大多数操作码。不支持的那就是与 SALS 认证有关。
```
Comment (majinghe) [inline docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst]:
```suggestion
ZooKeeper 代理过滤器解码 `Apache ZooKeeper <https://zookeeper.apache.org/>`_ 的客户端协议。它能够对负载中的请求、响应、事件进行解码。其中支持 `ZooKeeper 3.5 <https://github.com/apache/zookeeper/blob/master/zookeeper-server/src/main/java/org/apache/zookeeper/ZooDefs.java>`_  中已知的大多数操作码。不支持的那就是与 SALS 认证有关。
```

File: docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst [hunk 2]
```diff
@@ -40,132 +33,130 @@ in the configuration snippet below:
...
 .. _config_network_filters_zookeeper_proxy_stats:
 
-Statistics
-----------
+统计
+------
 
-Every configured ZooKeeper proxy filter has statistics rooted at *<stat_prefix>.zookeeper.*. The
-following counters are available:
+每一个ZooKeeper 代理过滤器的配置都有一个基于 *<stat_prefix>.zookeeper.* 的统计信息。以下的计数器可用：
```
Comment (majinghe) [inline docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst]:
```suggestion
每一个ZooKeeper 代理过滤器的配置都有一个基于 *<stat_prefix>.zookeeper.* 的统计信息。可用的计数器如下：
```
感觉这样能更通畅点。

**Comment by majinghe** [comment]
@vgbhfive 非常棒的翻译，一些comment 仅供参考。

**Review by addozhang — APPROVED** [review]
/LGTM

**Pull request merged**